api/analyticsd/client/client.go

```diff
@@ -0,0 +1,122 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/textileio/textile/v2/api/analyticsd/pb"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+type config struct {
+	active     bool
+	email      string
+	properties map[string]interface{}
+}
+
+type Option = func(*config)
+
+func WithActive() Option {
+	return func(c *config) {
+		c.active = true
+	}
+}
+
+func WithEmail(email string) Option {
+	return func(c *config) {
+		c.email = email
+	}
+}
+
+func WithProperties(properties map[string]interface{}) Option {
+	return func(c *config) {
+		c.properties = properties
+	}
+}
+
+type Client struct {
+	asc  pb.AnalyticsServiceClient
+	conn *grpc.ClientConn
+}
+
+func New(target string, opts ...grpc.DialOption) (*Client, error) {
+	conn, err := grpc.Dial(target, opts...)
+	if err != nil {
+		return nil, fmt.Errorf("creating gRPC client conn: %v", err)
+	}
+
+	c := pb.NewAnalyticsServiceClient(conn)
+
+	return &Client{
+		asc:  c,
+		conn: conn,
+	}, nil
+}
+
+func (c *Client) Identify(ctx context.Context, key string, accountType pb.AccountType, opts ...Option) error {
+	if c == nil {
+		return nil
+	}
```

Comment:
since these APIs only return `error` I added this noop if the client is `nil`. This way you can just call analytics client apis without having to check if the client was created or not.